contract-ui/tabs/nfts/components/upload-step.tsx

```diff
@@ -37,7 +37,7 @@ export const UploadStep: React.FC<UploadStepProps> = ({
               cursor="pointer"
               bg={noFile ? "red.200" : "inputBg"}
               _hover={{
-                bg: "inputBgHover",
+                bg: noFile ? "red.200" : "inputBgHover",
```

Comment:
is this true for *no* file or is it based on *there might be files but they are invalid?

Reply:
_just asking about var naming_

Reply:
it's actually `noFileOrOnlyInvalidFiles` but yeah